### DISCUSSION TITLE
eFORMS – Submitting notices with more than 500 lots

### DISCUSSION THREAD

**Discussion by AnrivaDeloitte** [OP-TED/eForms-SDK#1024] [Q&A]
Dear Publication Office,

On behalf of Lombardy Region (Italy) and through the national eSender ANAC, ARIA S.p.A. needs to submit a contract notice – general directive, standard regime – competition (Directive 2014/24/EU) composed of 3.000/3.500 lots for the supply of medicines.

Taking this into account, we would like to know whether there are any limitations in terms of the maximum of lots that can be published within one single notice and, if there are, how it is possible to cope with any given limitation.

In addition, we would like to inform you that any validation fails for notices containing more than 500 lots.

Thank you in advance and we look forward to hearing from you soon.
 
Kind regards,

Angelica Riva

**Comment by YvesJo** [comment]
Hi,
Thanks for sharing your concern.
To answer your question, you could have a look at our FAQ ([Are there any limitations in place for the TED API?](https://docs.ted.europa.eu/eforms-common/FAQ/index.html#:~:text=Are%20there%20any%20limitations%20in%20place%20for%20the%20TED%20API%3F) )
For large Result notices, our recommendation is to break them up into smaller notices; this may reduce the possibility of a timeout and also make the notice more legible for readers and end users.
As stated during [a previous workshop](https://op.europa.eu/en/web/ted-eforms/minutes_q2_20_june_2024#:~:text=As%20there%20are%20many%20factors%20at%20play%2C%20should%20you%20get%20a%20timeout%20on%20a%20notice%20you%20wish%20to%20publish%2C%20please%20contact%20us%20on%20a%20case%2Dby%2Dcase%20basis.%C2%A0), since there are many factors at play, should you get a timeout on a notice you wish to publish, please contact us on a case-by-case basis.
Alternatively, you can also send your request to the helpdesk.
KR

**Reply by AnrivaDeloitte** [reply]
Thank you @YvesJo for your reply.

As suggested above, we have sent a request to the helpdesk.

Since we need to publish a **contract notice** (notice type 16) for the supply of medicines made up of 1.250 lots by the first decade of October, we would like to know the exact number of lots it can contain and/or in how many smaller notices we should break it up.

Attached, you can find a sample of XML composed of just one lot filled with all the information we need to publish the contract notice described. Please note that this information is repeated for every other lot of the procedure.

With this information, we kindly ask you to let us know how many lots we can include in one contract notice avoiding the 3-minutes timeout as it is not specified in the FAQ.

Thanks a lot
AR
[notice-1.9-b7ab3399-8f82-4591-9a0d-42976ce9eede.zip](https://github.com/user-attachments/files/17116462/notice-1.9-b7ab3399-8f82-4591-9a0d-42976ce9eede.zip)

**Reply by YvesJo** [reply]
Hi Anriva,
Unfortunately it is not possible to specify an exact number of lots as the time required for validation is not just proportional to the number of lots, it also depends on the information specified per lot and with the number of defined organisations.
With SDK 1.10, the rules have been refactored to improve the performance, however I see in the provided example that SDK 1.9 is used.
I will try to get an indicative order of magnitude and will get back to you.
KR

**Reply by YvesJo** [reply]
When not mistaken, for SDK 1.9 , the largest number of lots ever submitted within a notice is 145. Please keep in mind that this is purely indicative and highly depends on the notice content (details per lots, number of organisations ...).

**Reply by AnrivaDeloitte** [reply]
Hi YvesJo,

thanks for your help. Do you have any updates for our concern?

I confirm that ARIA S.p.A., on behalf of Lombardy Region, needs to submit a 1.250-lots notice with SDK 1.9 for the supply of medicines.

In particular, this note will include 2 organisations and all the information per lot you can find in the attached file.

Thanks a lot.
[E16_f701dcd7-479d-433d-83af-833ab3ebd53e_01_Validato.zip](https://github.com/user-attachments/files/17191448/E16_f701dcd7-479d-433d-83af-833ab3ebd53e_01_Validato.zip)


AR

**Reply by YvesJo** [reply]
Hi,
Please be aware that this public communication channel is exclusively for SDK related concerns and shouldn't be used as an alternate contact point for the helpdesk.
The helpdesk is already processing your request and you should have received an answer from them, therefore I will close this discussion.
KR
